src/electionguard-ui/ElectionGuard.UI/ViewModels/ViewTallyViewModel.cs

```diff
@@ -65,6 +66,7 @@ public partial class ViewTallyViewModel : BaseViewModel
         _decryptionShareService = decryptionShareService;
         _challengeResponseService = challengeResponseService;
         _plaintextTallyService = plaintextTallyService;
+        _logger = logger;

```

Comment:
insert generic notes about logging and SRP.

Reply:
This has been discussed and will be reviewed in the future